website/documentation/contribute/_index.md

```diff
@@ -37,7 +37,7 @@ Gardener uses GitHub to manage reviews of pull requests.
 
 * If you have a trivial fix or improvement, go ahead and create a pull request,
   addressing (with `@...`) a suitable maintainer of this repository (see
-  [CODEOWNERS](https://github.com/gardener/gardener/blob/master/CODEOWNERS) of the
+  [OWNERS](https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md) of the
```

Comment:
If we don't maintain owners, I propose to drop the `addressing (with @...) a suitable maintainer of this repository (see ...`
part. Referencing external docu does not help here